src/bvh/hash.hpp

```diff
@@ -36,14 +36,12 @@
 #include <cstdint>
 #include "math/vec.hpp"
 #include "kdop.hpp"
+#include "util/kokkos.hpp"
 
-#if !defined(BVH_ENABLE_KOKKOS) || defined(KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HOST)
+#if defined(KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HOST)
```

Comment:
`KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HOST` was removed a while ago and I guess I forgot to remove it from here. I think the best bet is to change this to `defined(__GNUC__)` since I believe that will work for both gcc and clang host compilers.

Reply:
Replaced with `__GNUC__` 👌